docs/user-guide/commands/argocd.md

```diff
@@ -0,0 +1,50 @@
+## argocd
```

Comment:
Add link docs/user-guide/commands/argocd.md to https://github.com/argoproj/argo-cd/blob/90227f226d38b3e43090e56f62d8e9baa29a3c5a/mkdocs.yml#L87